### PR TITLE
Merge polygons

### PR DESCRIPTION
**Pull request by frodrigo**
POC for polygons support in merge analyser.

### CODE REVIEW

File: analysers/analyser_merge_forest_compartment_FR.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+class Analyser_Merge_Forest_Compartment_FR(Analyser_Merge_Point):
+    requires_tables_common = ['polygons']
+
+    def __init__(self, config, logger = None):
+        Analyser_Merge_Point.__init__(self, config, logger)
+        self.def_class_missing_official(item = 8550, id = 1, level = 3, tags = ['merge', 'boundary', 'fix:survey', 'fix:imagery'],
+            title = T_('Forest compartment not integrated'),
+            fix = T_('Please use the geopf.fr WMTS FORETS.PUBLIQUES service or the ONF data at http://metadata.carmencarto.fr/geonetwork/105/api/records/fr-662043116-82880F0D-E1C4-4EF3-80AF-416977F118F1 to create the appropriate boundary with forest compartment tags in OSM'),
+            trap = T_('Do not import to OSM the POI created by Osmose, which is just there to help you locate the missing forest compartment'))
```
Comment (Famlam) [inline analysers/analyser_merge_forest_compartment_FR.py:35]:
```suggestion
            fix = T_('Please use the geopf.fr WMTS FORETS.PUBLIQUES service or the ONF data at http://metadata.carmencarto.fr/geonetwork/105/api/records/fr-662043116-82880F0D-E1C4-4EF3-80AF-416977F118F1 to create the appropriate boundary with forest compartment tags in OSM.'),
            trap = T_('Do not import the POI created by Osmose to OSM. It is just there to help you locate the missing forest compartment.'))
```

**Review by Famlam — COMMENTED** [review]
I'm not experienced enough with Merge analysers to give a full review, but at least I can review the text :)

**Comment by frodrigo** [comment]
@StC-OSM This PR will replace your. It uses 'polygon' in place of 'way'. The contend is linted. Can you review it, please.

**Comment by Famlam** [comment]
> Also, shouldn't the analyser be added to the config?

@frodrigo Isn't this necessary?

**Comment by frodrigo** [comment]
> > Also, shouldn't the analyser be added to the config?
> 
> @frodrigo Isn't this necessary?

Right ! Done 79b1b0a6

**Comment by frodrigo** [comment]
Update in progress
https://osmose.openstreetmap.fr/fr/issues/open?item=8550&source=&class=&useDevItem=all&username=&bbox=

**Pull request merged**